Week3/homework/step2-3.js

```diff
@@ -5,9 +5,12 @@ function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  if (num <= 0) {
+    return result;
+  }
```

Comment:
This test is redundant and can be removed. The `for` loop body will not be executed if `num <= 0`. This is because the loop condition is tested at the start of the loop.